client/src/components/Tabs/Tabs.test.tsx

```diff
@@ -2,14 +2,16 @@ import { render } from '@testing-library/react';
 import React from 'react';
 import Tabs from './Tabs';
 
-describe('Tabs tests', () => {
-  it('should display buttons for the given tabs', () => {
-    // `render` renders the component into document.body and returns an object with lots of query functions
-    const { getByRole } = render(<Tabs tabs={['First tab', 'Second tab']} />);
+// TODO: fix test later
```

Comment:
Does this test work or are the tabs broken? Could get rid of this file for now?

Reply:
We weren't passing all the required props to Tabs but i'll remove the file

Reply:
Agree, remove the file. This was just a 'hello world' to try out Jest.